DESCRIPTION

```diff
@@ -14,7 +14,6 @@ URL: https://github.com/metrumresearchgroup/pmplots/
 BugReports: https://github.com/metrumresearchgroup/pmplots/issues
 Encoding: UTF-8
 Language: en-US
-LazyData: true
```

Comment:
what was the reason for removing lazydata?

Reply:
No official / registered data sets.